.github/workflows/github-repo-stats.yml

```diff
@@ -0,0 +1,19 @@
+name: github-repo-stats
+
+on:
+  schedule:
+    # Run this once per day, towards the end of the day for keeping the most
+    # recent data point most meaningful (hours are interpreted in UTC).
+    - cron: '0 23 * * *'
```

Comment:
This is 3pm PDT - what if we did 01:00 or 02:00 UTC instead to capture the end of the day all the way over in pacific time?